<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LegendSettingsWidget</class>
 <widget class="QWidget" name="LegendSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>398</width>
    <height>92</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Legend position</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="legendPositionComboBox"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Align</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="legendAlignComboBox"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="applyToAllCheckBox">
     <property name="text">
      <string>Apply to all graphics</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
